src/pages/find-offers/FindOffers.jsx

```diff
@@ -1,7 +1,19 @@
 import PageWrapper from '~/components/page-wrapper/PageWrapper'
+import Stack from '@mui/material/Stack'
+import AppViewSwitcher from '~/components/app-view-switcher/AppViewSwitcher'
+import { useState } from 'react'
 
 const FindOffers = () => {
-  return <PageWrapper>Find offers</PageWrapper>
+  const [view, setView] = useState('list')
```

Comment:
Don't you think it's better to save this value in searchParams so it will be saved when the page reloads

Reply:
I forgot about it. I will put it in searchParams in next commit